assemblies/core/static/src/main/resources/classes/log4j2.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ===================================================================== -->
+<!--                                                                       -->
+<!--  Log4j2 Configuration                                                  -->
+<!--                                                                       -->
+<!-- ===================================================================== -->
+
+<!--
+   | For more configuration information and examples see the Apache Log4j
+   | website: https://logging.apache.org/log4j/2.x/manual/configuration.html
+ -->
+
+<Configuration name="PentahoKettle" status="INFO" dest="out">
+
+    <Appenders>
+
+        <!-- ================================= -->
+        <!-- Preserve messages in a local file -->
+        <!-- ================================= -->
+
+        <!-- A time/date based rolling appender -->
+        <RollingFile name="PENTAHOFILE" fileName="logs/pdi.log" filePattern="logs/pdi.log.%d{yyyy-MM-dd}" append="false" bufferedIO="true" immediateFlush="true">
+            <PatternLayout>
+                <Pattern>%d %-5p [%c] %m%n</Pattern>
+            </PatternLayout>
+            <Policies>
+                <OnStartupTriggeringPolicy />
+                <SizeBasedTriggeringPolicy size="20 MB" />
+                <TimeBasedTriggeringPolicy />
+            </Policies>
+            <DefaultRolloverStrategy max="1" fileIndex="min"/>
+        </RollingFile>
+
+
+        <!-- ================== -->
+        <!-- Log4JDBC Appenders -->
+        <!-- ================== -->
+        <File name="sql-appender" fileName="logs/log4jdbc-sql.log" append="false">
```

Comment:
It looks like sql-appender and the following 3 appenders were also commented out before in log4j.xml